pkg/generator_test.go

```diff
@@ -246,6 +246,39 @@ func (_m *mockRequester_unexported) Get() {
 	)
 }
 
+func (s *GeneratorSuite) TestGeneratorExportedOnFlag() {
+	expected := `// MockRequester_unexported is an autogenerated mock type for the requester_unexported type
+type MockRequester_unexported struct {
+	mock.Mock
+}
+
+// Get provides a mock function with given fields:
+func (_m *MockRequester_unexported) Get() {
+	_m.Called()
+}
+`
+	generator := NewGenerator(
```

Comment:
Note, that `checkGeneration` accepts some of the configs, but not all. Instead of introducing another boolean argument for the function I decided to create a generator manually.